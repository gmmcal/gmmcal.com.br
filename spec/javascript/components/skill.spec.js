import { mount } from '@vue/test-utils'
import Skill from '@/components/skill'
import create from '../settings/utils'

describe('Skill', () => {
  let props, wrapper

  beforeEach(() => {
    props = {
      propsData: {
        skill: create('skill')
      }
    }

    wrapper = mount(Skill, props)
  })

  it('is a Vue instance', () => {
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  it('has data properties', () => {
    const expected = []
    const received = Object.keys(wrapper.vm.$data)
    expect(received).toEqual(expected)
  })

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
