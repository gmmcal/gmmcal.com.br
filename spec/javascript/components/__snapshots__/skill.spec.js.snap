// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skill renders correctly 1`] = `
<div class="col-xs-12 col-sm-4 col-md-4 col-lg-4 skills-area">
  <div class="col-xs-12 col-sm-6 col-md-6 col-lg-6 skills"><span data-percent="100" class="chart skill-bg"><span class="percent">100</span>
    <div></div>
    </span>
    <h4>Skill Name</h4>
    <p>Skill hero</p>
  </div>
</div>
`;
